docs/source/private-data/private-data.md

```diff
@@ -151,6 +151,15 @@ documentation on [transaction flow](../txflow.html).
    their copy of the private state database and private writeset storage. The
    private data is then deleted from the `transient data store`.
 
+The [Fabric Gateway](../gateway.html) takes care of collecting the required set of endorsements
+on behalf of the client application.  It does this by running the first simulation on a peer in
```

Comment:
The gateway peer is a peer in the gateway's org., would it not run the simulation on itself.

Reply:
Is it debatable if we need to re-document what simulation means in describing gateway? ... 
For example, is, "The gateway peer simulates the transaction and aggregates the relevant endorsement policies generated by the transaction chaincode." -- which goes less into the weeds --  satisfactory and preferable or the reverse.

Reply:
Yes it could do.  It will select the peer with the largest ledger block height in the org.